pages/common/sort.md

```diff
@@ -13,3 +13,11 @@
 - Sort passwd file by the 3rd field:
 
 `sort -t: -k 3n /etc/passwd`
+
+- Sort a file as number in ascending order:
+
+`sort -n {{filename}}`
+
+- Sort a file and delete duplicate lines:
```

Comment:
It would be nice to use the word "unique" here to provide a mnemonic for the option name -u.